frontend/awx/resources/inventories/inventoryHostsPage/inventoryHostsGroups.cy.tsx

```diff
@@ -0,0 +1,239 @@
+    });
+
+    it('Filter Host Groups by created by', () => {
+      cy.mount(<InventoryHostsGroups />, {
+        path: '/inventories/:inventory_type/:id/hosts/:host_id/*',
+        initialEntries: ['/inventories/inventory/1/hosts/1/groups'],
+      });
+      cy.intercept(
+        {
+          method: 'GET',
+          url: '/api/v2/hosts/*/all_groups?created_by__username__icontains=test*',
+        },
+        {
+          fixture: 'group.json',
+        }
+      ).as('createdByFilter');
+      cy.filterTableByTypeAndText(/^Created by$/, 'test');
+      cy.get('@createdByFilter.all').should('have.length.least', 1);
+      cy.clearAllFilters();
+    });
+
+    it('Filter Host Groups by modified by', () => {
+      cy.mount(<InventoryHostsGroups />, {
+        path: '/inventories/:inventory_type/:id/hosts/:host_id/*',
+        initialEntries: ['/inventories/inventory/1/hosts/1/groups'],
+      });
+      cy.intercept(
+        {
+          method: 'GET',
+          url: '/api/v2/hosts/*/all_groups?modified_by__username__icontains=test*',
+        },
+        {
+          fixture: 'group.json',
+        }
+      ).as('modifiedByFilter');
+      cy.filterTableByTypeAndText(/^Modified by$/, 'test');
+      cy.get('@modifiedByFilter.all').should('have.length.least', 1);
+      cy.clearAllFilters();
+    });
+
+    it('Add group button is enabled if the user has permission to add a group', () => {
+      cy.mount(<InventoryHostsGroups />, {
+        path: '/inventories/:inventory_type/:id/hosts/:host_id/*',
+        initialEntries: ['/inventories/inventory/1/hosts/1/groups'],
+      });
+      cy.contains('button', /^Add$/).should('have.attr', 'aria-disabled', 'false');
+    });
+
+    it('Edit inventory group row action is enabled if the user has permission to edit inventory group', () => {
+      cy.mount(<InventoryHostsGroups />, {
+        path: '/inventories/:inventory_type/:id/hosts/:host_id/*',
+        initialEntries: ['/inventories/inventory/1/hosts/1/groups'],
+      });
+      cy.contains('tr', 'Test groups 1').within(() => {
+        cy.get('[data-cy="actions-column-cell"]').within(() => {
+          cy.get(`[data-cy="edit-group"]`).should('have.attr', 'aria-disabled', 'false');
+        });
+      });
+    });
+
+    it('Add group button is disabled if the user does not have permission to associate a host with a group', () => {
+      cy.stub(useOptions, 'useOptions').callsFake(() => ({
+        data: {
+          actions: {},
+        },
+      }));
+      cy.mount(<InventoryHostsGroups />, {
+        path: '/inventories/:inventory_type/:id/hosts/:host_id/*',
+        initialEntries: ['/inventories/inventory/1/hosts/1/groups'],
+      });
+      cy.contains('button', /^Add$/).should('have.attr', 'aria-disabled', 'true');
+    });
+
+    it('Edit inventory group row action is disabled if the user does not have permission to edit inventory group', () => {
+      cy.fixture('groups')
+        .then((inventoryGroups) => {
```

Comment:
Making this change will allow you to remove the commented out lines for the linter at the top of the file.
```suggestion
        .then((inventoryGroups:AwxItemsResponse<InventoryGroup) => {
```

Reply:
done